website/docs/choosing-a-communication.md

```diff
@@ -15,17 +15,17 @@ The `PortalsPlugin` class is part of the Portals library and provides a helpful,
 
 We recommend this approach when performing small UI tasks like dismissing a native modal from inside the web application, or passing short messages. This mechanism may also be more appealing to small teams or solo developers who want to communicate through a Portal without the desire to build a Custom Plugin.
 
-See our guide on [How To Use The PortalsPlugin](./how-to/using-the-portals-plugin).
+See our guide on [How To Use The PortalsPlugin](./).
```

Comment:
It's generally safer and more convenient to link to the markdown document rather than a URL. One benefit is that the link will work on GitHub. Docusaurus will automagically convert the link to the correct URL.

```suggestion
See our guide on [How To Use The PortalsPlugin](overview.md).
```

Reply:
Pretty sure these links were broken by accident.